### PR TITLE
added sources D-G

### PR DESCRIPTION
**Pull request by thja**

### CODE REVIEW

**Comment by thja** [comment]
I signed it!

File: organizations/oath.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "OATH",
+  "name": "Oath",
+  "orgUrl": "https://www.oath.com/"
```
Comment (pfrisella) [inline organizations/oath.json]: by chance do you have an iconUrl?

File: sources/clever_reach.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/clever_reach.json]: filename should match Id: cleverreach.json

File: organizations/clever_reach.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/clever_reach.json]: filename should match id: cleverreach.json

File: sources/etracker.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "ETRACKER",
```
Comment (pfrisella) [inline sources/etracker.json]: etracker Analytics ? scoping to a particular subset of data is preferred if applicable

File: sources/facebook_business_invoices.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "FACEBOOK_BUSINESS_INVOICES",
+  "name": "Facebook Business Invoices",
+  "categories": ["SOCIAL"],
+  "organization": "FACEBOOK",
+  "iconUrl": "https://s3.amazonaws.com/Supermetrics/images/dsLogos/FB.png",
+  "sourceUrl": "https://developers.facebook.com/docs/marketing-api/business-manager-api",
```
Comment (pfrisella) [inline sources/facebook_business_invoices.json]: https://business.facebook.com/  ??

File: sources/getstat.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/getstat.json]: file: stat_search_analytics.json

File: sources/getstat.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "STAT",
```
Comment (pfrisella) [inline sources/getstat.json]: STAT_SEARCH_ANALYTICS

File: sources/getstat.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "STAT Search Analytics",
+  "categories": ["SEO"],
+  "organization": "STAT",
```
Comment (pfrisella) [inline sources/getstat.json]: STAT_SEARCH_ANALYTICS

File: organizations/stat.json [hunk 9]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "STAT",
```
Comment (pfrisella) [inline organizations/stat.json]: STAT_SEARCH_ANALYTICS

File: organizations/stat.json [hunk 10]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/stat.json]: file: stat_search_analytics.json

**Comment by thja** [comment]
@pfrisella please review again

**Pull request merged**